eng/config/BannedSymbols.txt

```diff
@@ -8,6 +8,8 @@ T:Microsoft.CodeAnalysis.Formatting.FormattingOptions; Use FormattingOptions2 in
 T:Microsoft.CodeAnalysis.Options.PerLanguageOption`1; Use PerLanguageOption2<T> instead
 T:Microsoft.CodeAnalysis.Options.Option`1; Use 'Microsoft.CodeAnalysis.Options.Option2' instead
 P:Microsoft.CodeAnalysis.Completion.CompletionContext.Options; Use CompletionOptions instead.
+P:Microsoft.CodeAnalysis.Completion.CompletionItem.Properties; Use GetProperties instead.
```

Comment:
did you mean 'GetProperty' here?

Reply:
GetProperties was intended. It returns the ImmutableArray which could be enumerated if that is the intent of the caller.